src/client/types.ts

```diff
@@ -0,0 +1,22 @@
+export interface Feature {
```

Comment:
These types were referenced used in the original client side PR (https://github.com/Automattic/wp-feature-api/pull/9), but this file did not seem to exist.